recipe/meta.yaml

```diff
@@ -29,21 +29,21 @@ requirements:
   host:
     - python
     - pip
-    - conda-package-handling >=1.1
+    - conda-package-handling >=1.3
   run:
     - beautifulsoup4
     - chardet
-    - conda  >=4.3
+    - conda  >=4.5
     - contextlib2        # [py27]
     - enum34             # [py27]
     - filelock
     - futures            # [py27]
     - jinja2
-    - patchelf           # [linux]
+    - patchelf      # [linux]
     - pkginfo
     - psutil
     - python
-    - py-lief            # [not (win and py2k)]
+    - py-lief  # [not (win and py2k) and not ppc64le]
```

Comment:
`py-lief` is a hard dependency on `ppc64le` due to https://github.com/conda/conda-build/issues/3514#issuecomment-495077976